tests/test_csr.py

```diff
@@ -451,3 +449,44 @@ def test_saving_public_certifate_disables_signing_request_fields(admin_client):
     # csr, public_certificate, common_name, organization_name, country_name,
     # state_or_province_name, locality_name, email_address, certificate
     assert response.content.decode().count("readonly") == 9
+
+
+@pytest.mark.django_db
+def test_csr_display(admin_client, client):
+    add_url = reverse("admin:simple_certmanager_signingrequest_add")
+
+    response = admin_client.get(add_url)
+
+    assert response.status_code == 200
+    assert "CSR not generated yet" in response.content.decode()
+
+    data = {
+        "common_name": "test.com",
+        "country_name": "US",
+        "organization_name": "Test Org",
+        "state_or_province_name": "Test State",
+        "email_address": "test@test.com",
+    }
+
+    response = admin_client.post(add_url, data, follow=True)
+
+    assert response.status_code == 200
+
+    signing_request = SigningRequest.objects.get()
+    assert signing_request is not None
```

Comment:
The default `django.db.models.Manager.get()` raises an exception, it never returns None.